### ISSUE TITLE
HDR Cohort Discovery - Heatbeat Support

### ISSUE THREAD

**Issue by thomgiles** [Health-Informatics-UoN/hutch-bunny#7]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Is your proposal related to a problem or functionality gap?

Link now supports heartbeats Can we do this too? 

### Describe your proposal

POST
<Rquest>/link_connector_api/statistics-controller/heartBeat

application/json
 
{
  "collection": "string",
  "dbActive": true,
  "diskInfo": {
    "totalSpace": "string",
    "unallocated": "string",
    "usableSpace": "string"
  },
  "genedActive": true,
  "interval": 0,
  "linkIp": "string",
  "memoryInfo": {
    "available": "string",
    "swapTotal": "string",
    "swapUsed": "string",
    "total": "string",
    "used": "string"
  },
  "processorInfo": {
    "logicalProcessorCount": 0,
    "physicalProcessorCount": 0,
    "processorIdentifier": "string",
    "usage": "string"
  },
  "time": "2024-12-02T10:31:14.319Z"
}

### Describe alternatives you've considered

N/A

### Are you willing to contribute to developing this feature?

None

**Comment by prquinlan** [comment]
Adding note from conversion. This should be an "opt-in" feature and clearly described based on the data it sends and makes visible. I am not sure everyone will want this enabled.

**Comment by AndyRae** [comment]
Looking again at this - given the UI of RQUEST isn't populated, and no RQUEST behaviour seems to be impacted either. 

I'm not sure it's worth implementing this at all?

**Comment by prquinlan** [comment]
Agree, I will close this for now, if there a reason we can capture
